source/jquery.flot.js

```diff
@@ -984,6 +987,25 @@ Licensed under the MIT license.
             executeHooks(hooks.bindEvents, [eventHolder]);
         }
 
+        function addEventHandler(event, handler, eventHolder, priority) {
+            var key = eventHolder + event;
+            var eventlist = eventManager[key];
+            if (eventlist === undefined) {
+                eventlist = [];
+            }
+
+            // var newEventList = [];
+            eventlist.push({"event": event, "handler": handler, "eventHolder": eventHolder, "priority": priority});
+            eventlist.sort(function(a, b) { return b.priority - a.priority });
+            for (var i = 0; i < eventlist.length; i++) {
+                var eventData = eventlist[i];
+                eventData.eventHolder.unbind(eventData.event, eventData.handler);
+                eventData.eventHolder.bind(eventData.event, eventData.handler);
```

Comment:
(So the purpose of this is to (re-)register each event handler to happen in the new sorted order?)

Reply:
Yes.